publisher/src/main/java/com/pavlovmedia/oss/jaxrs/publisher/command/ProviderCommands.java

```diff
@@ -14,60 +14,66 @@
  * limitations under the License.
  */
 package com.pavlovmedia.oss.jaxrs.publisher.command;
+ 
 
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Properties;
-import org.apache.felix.scr.annotations.Property;
-import org.apache.felix.scr.annotations.Reference;
-import org.apache.felix.scr.annotations.Service;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.log.Logger;
+import org.osgi.service.log.LoggerFactory;
+import org.osgi.service.metatype.annotations.Designate;
 
 import com.pavlovmedia.oss.jaxrs.publisher.api.Publisher;
+import com.pavlovmedia.oss.jaxrs.publisher.impl.config.ProviderCommandConfig;
 
 /**
  * Commands to facilitate debugging of the JAX-RS system.
  * 
  * @author Shawn Dempsay {@literal <sdempsay@pavlovmedia.com>}
  *
  */
-@Component
-@Service(ProviderCommands.class)
-@Properties({ 
-    @Property(name = "osgi.command.scope", value = "jax"),
-    @Property(name = "osgi.command.function", value = { "getEndpoints", "getFeatures", "getProviders" }),
-    @Property(name = Publisher.SCAN_IGNORE, value = "true")
```

Comment:
Do these command still work from the gogo shell now?

Reply:
These commands seem to work. I get the following output - 
<img width="582" alt="gogo provider commands" src="https://user-images.githubusercontent.com/54722029/66962192-02f4ab00-f036-11e9-8537-1271917bea32.png">